.github/settings.yml

```diff
@@ -123,15 +123,15 @@ branches:
       # Required. Require status checks to pass before merging. Set to null to disable
       required_status_checks:
         # Required. Require branches to be up to date before merging.
-        strict: true
+        strict: false
         # Required. The list of status checks to require in order to merge into this branch
-        contexts: [ "bors" ]
```

Comment:
I'm not sure what to put here

Reply:
Could probably leave this as null for now and revisit it once CI has been stable for a bit.